src/app/components/Figure/index.jsx

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
 import { string } from 'prop-types';
+import styled from 'styled-components';
 import Caption from './Caption';
+import { GEL_SPACING_DBL } from '../../lib/constants/styles';
 
 const renderCaption = caption =>
   caption ? <Caption>{caption}</Caption> : null;
 
+const StyledFigure = styled.figure`
+  margin: 0;
+  padding: 0 0 ${GEL_SPACING_DBL} 0;
```

Comment:
It is simpler and more concise to just set `padding-bottom` here.

Reply:
@jtart I suggested to have this form of padding, since it resets the padding top & left and right to 0.